src/Sepes.RestApi.IntegrationTests/Setup/Scenarios/AzureServicesMockedAndSucceeding.cs

```diff
@@ -13,11 +13,13 @@ public void RegisterServices(IServiceCollection serviceCollection)
         {
             //NON-AZURE, BUT EXTERNAL
             serviceCollection.SwapTransient<IWbsApiService>(provider => WbsApiMockServiceFactory.GetService(true, false));
-
             serviceCollection.SwapTransient<IAzureRoleAssignmentService>(provider => AzureRoleAssignmentMockServiceFactory.CreateBasicForCreate().Object);
             serviceCollection.SwapTransient<IAzureKeyVaultSecretService>(provider => AzureKeyVaultSecretMockServiceFactory.CreateBasicForResourceCreate().Object);
+            serviceCollection.SwapTransient<IAzureResourceGroupService>(provider => AzureResourceGroupMockServiceFactory.CreateBasicForCreate().Object); 
+            
```

Comment:
So these creates mock services for integration tests? Or?

Reply:
True